src/syscheckd/db/src/registry.cpp

```diff
@@ -5,9 +5,12 @@
  *
  * @copyright Copyright (C) 2015-2021 Wazuh, Inc.
  */
+#ifdef __cplusplus
+extern "C" {
+#endif
 
 #ifdef WIN32
```

Comment:
We could remove this `#ifdef`, the new CMakelists file ensures that this file won't be compiled on systems that are not Windows.